src/context.rs

```diff
@@ -1159,6 +1159,96 @@ impl Context {
             Err(ret)
         }
     }
+
+    /// Cause the policy to include a signed authorization
+    #[allow(clippy::too_many_arguments)]
+    pub fn policy_signed(
+        &mut self,
+        policy_session: Session,
+        auth_handle: ObjectHandle,
```

Comment:
Is there a reason for auth_handle to not be an AuthHandle?

Reply:
Yep. That I was being silly! Will fix it.

Reply:
Oh, right. No, that wasn't silly, I was just wrong at naming. Because it's not an `auth_handle` but an `auth_object`.

Reply:
I fixed the object name here to `auth_object`, and fixed the type on `policy_secret` to `AuthHandle`.